apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -2164,7 +2164,7 @@ SPEC CHECKSUMS:
   ReactCommon: f7da14a8827b72704169a48c929bcde802698361
```

Comment:
Contact screen is broken

<img width="350" alt="image" src="https://github.com/user-attachments/assets/fdceabfd-e3c4-413b-a1b7-ef907838283f">